multi-node/aws/pkg/config/templates/cloud-config-etcd

```diff
@@ -61,7 +61,7 @@ coreos:
         [Service]
         Type=oneshot
         RemainAfterExit=yes
-        ExecStart=/opt/bin/ext4-format-volume-once /dev/xvdf
+        ExecStart=/usr/sbin/mkfs.ext4 -p /dev/xvdf
```

Comment:
disregard my previous comment- after talking to crawford I've reverted to first checking with `blkid`, and then leaving the `-p` arg to `mkfs` for extra safety.